XenoHolographic-it/IAR-SDK-Sample-Android
# Feature/IAR-4920

# Overview

Add NFC-write screen and implement write NFC functionality

# Resources

https://iversoft.atlassian.net/browse/IAR-4920

# Technical Considerations

-

# Known Issues

-

# Testing Instructions

* Enter marker id in text field, click ‘Write Tag to NFC Device’ button, scan a NFC tag, to see if there is ‘Write NFC successfully” message and the correct marker id.
* Click ‘Randomize ID” button, the random generated Id should show in text field. Scan a NFC tag, to see if there is ‘Write NFC successfully” message and the correct marker id.

# Screenshots
![Screenshot_4920-1](https://user-images.githubusercontent.com/96147375/152170935-aa9d45e4-9e16-49a1-870c-aff5f822d301.jpg)
![Screenshot_4920-2](https://user-images.githubusercontent.com/96147375/152170994-9e979699-d83a-400a-a86e-473f67b766f3.jpg)
